test-data/unit/check-kwargs.test

```diff
@@ -485,7 +562,7 @@ def g(arg: int = 0, **kwargs: object) -> None:
 
 d = {} # type: Dict[str, object]
 f(**d)
-g(**d)  # E: Argument 1 to "g" has incompatible type "**Dict[str, object]"; expected "int"
+g(**d)
 
 m = {} # type: Mapping[str, object]
 f(**m)
```

Comment:
On the line below there is a `# TODO: should be an error`
Presumably it shouldn't be anymore?